django_elasticsearch_dsl/documents.py

```diff
@@ -206,7 +207,12 @@ def _get_actions(self, object_list, action):
             paginator = Paginator(
                 object_list, self._doc_type.queryset_pagination
             )
+            number_of_pages = len(paginator.page_range)
+            current_page = 0
             for page in paginator.page_range:
+                current_page += 1
+                print("NUMBER OF PAGES: " + str(number_of_pages))
```

Comment:
If I may offer some criticism: Maybe you would like to remove the `print` statements before this gets merged?